Utils/spark-localrun-mock/test/java/com/microsoft/azure/hdinsight/spark/common/JvmProcess.kt

```diff
@@ -23,21 +23,20 @@
 package com.microsoft.azure.hdinsight.spark.common
 
 import java.io.File
-import java.nio.file.Paths
 import java.util.*
 import kotlin.collections.HashMap
 
 open class JvmProcess {
     open var workingDirectory: String = System.getProperty("user.dir")
 
-    var classpath: String = System.getProperty("java.class.path")
+    private var classpath: String = System.getProperty("java.class.path")
 
-    var jvm: String = Paths.get(System.getProperty("java.home"), "bin", "java").toString()
+    private var jvm: String = File(System.getProperty("java.home"), "bin/java").path
```

Comment:
Paths -> File change is required otherwise it will fall into a build error(a suspected bug in java.nio.file) **java.lang.NoClassDefFoundError: Could not initialize class java.nio.file.FileSystems$DefaultFileSystemHolder**

Reply:
It's weird...